soc/xtensa/intel_adsp/ace/sram.c

```diff
@@ -40,10 +38,15 @@ __imr void lp_sram_init(void)
 {
 	uint32_t lpsram_ebb_quantity = ace_lpsram_get_bank_count();
 	volatile uint32_t *l2usbpmptr = (volatile uint32_t *)ACE_L2MM->l2usbpmptr;
+	volatile uint8_t *status = (volatile uint8_t *)l2usbpmptr + 4;
+	int idx;
```

Comment:
for the sake of correctness this should be `uint32_t` like `lpsram_ebb_quantity`